// Commit 3da0976a8190211e43e285896bfb6dcd80dd7d50 breaks FBdev video, so we need to revert it

--- a/Source/Core/VideoCommon/PixelShaderGen.cpp
+++ b/Source/Core/VideoCommon/PixelShaderGen.cpp
@@ -371,15 +371,6 @@
             "int3 iround(float3 x) {{ return int3(round(x)); }}\n"
             "int4 iround(float4 x) {{ return int4(round(x)); }}\n\n");
 
-  // GLSL's any() and all() only accept vector types, while HLSL's also accept scalar types. We're
-  // adding these for convenience because while vector comparisons return a bool scalar in GLSL,
-  // allowing the results to be used directly in an if statement, they return a bool vector in HLSL,
-  // necessitating the use of any() or all() to reduce it to a scalar.
-  if (api_type == APIType::OpenGL || api_type == APIType::Vulkan)
-  {
-    out.Write("bool any(bool b) {{ return b; }}\n"
-              "bool all(bool b) {{ return b; }}\n\n");
-  }
 
   if (api_type == APIType::OpenGL || api_type == APIType::Vulkan)
   {
